src/plugins/controls/public/control_group/control_group_strings.ts

```diff
@@ -41,30 +41,73 @@ export const ControlGroupStrings = {
+        i18n.translate('controls.controlGroup.manageControl.dataSource.formGroupTitle', {
+          defaultMessage: 'Data source',
+        }),
+      getFormGroupDescription: () =>
+        i18n.translate('controls.controlGroup.manageControl.dataSource.formGroupDescription', {
+          defaultMessage: 'Select the data view and field that you want to create a control for.',
+        }),
+      getSelectDataViewMessage: () =>
+        i18n.translate('controls.controlGroup.manageControl.dataSource.selectDataViewMessage', {
+          defaultMessage: 'Please select a data view',
+        }),
+      getDataViewTitle: () =>
+        i18n.translate('controls.controlGroup.manageControl.dataSource.dataViewTitle', {
+          defaultMessage: 'Data view',
+        }),
+      getSelectFieldMessage: () =>
+        i18n.translate('controls.controlGroup.manageControl.dataSource.selectFieldMessage', {
+          defaultMessage: 'Please select a field',
+        }),
+      getFieldTitle: () =>
+        i18n.translate('controls.controlGroup.manageControl.dataSource.fieldTitle', {
+          defaultMessage: 'Field',
+        }),
+      getControlTypeTitle: () =>
+        i18n.translate('controls.controlGroup.manageControl.dataSource.controlTypesTitle', {
+          defaultMessage: 'Control type',
+        }),
+    },
+    displaySettings: {
+      getFormGroupTitle: () =>
+        i18n.translate('controls.controlGroup.manageControl.displaySettings.formGroupTitle', {
+          defaultMessage: 'Display settings',
+        }),
+      getFormGroupDescription: () =>
+        i18n.translate('controls.controlGroup.manageControl.displaySettings.formGroupDescription', {
+          defaultMessage: 'Change how the control will appear on your dashboard.',
+        }),
+      getTitleInputTitle: () =>
+        i18n.translate('controls.controlGroup.manageControl.displaySettings.titleInputTitle', {
+          defaultMessage: 'Label',
+        }),
+      getWidthInputTitle: () =>
+        i18n.translate('controls.controlGroup.manageControl.displaySettings.widthInputTitle', {
+          defaultMessage: 'Minimum width',
+        }),
+      getGrowSwitchTitle: () =>
+        i18n.translate('controls.controlGroup.manageControl.displaySettings.growSwitchTitle', {
+          defaultMessage: 'Expand width to fit available space',
+        }),
+    },
+    controlTypeSettings: {
+      getFormGroupTitle: (type: string) =>
+        i18n.translate('controls.controlGroup.manageControl.controlTypeSettings.formGroupTitle', {
+          defaultMessage: '{controlType} settings',
+          values: { controlType: type },
+        }),
+      getFormGroupDescription: (type: string) =>
+        i18n.translate(
+          'controls.controlGroup.manageControl.controlTypeSettings.formGroupDescription',
+          {
+            defaultMessage: 'Custom settings for your {controlType} control.',
+            values: { controlType: type.toLocaleLowerCase() },
+          }
+        ),
+    },
```

Comment:
Copy reference:

| Creating control | Editing control |
|--------|-------|
| ![image](https://github.com/elastic/kibana/assets/8698078/b86e8171-6d7b-4c9d-90e2-412389494aea) | ![image](https://github.com/elastic/kibana/assets/8698078/ed23e681-5812-4c48-b602-59260863862d) |

Note that only the title of the flyout is currently changing between create and edit mode. Should the new copy also be changed in these different scenarios? @elastic/ux-writers

Reply:
Hi! Reviewing on behalf of Gail:
- To your question, it's fine that it's just the title that changes. "create" in the description of the first section is still valid to me because if you change the data view or the field, it'll still add the control there.
- I'd replace "Please select a field" with "No field selected yet", since this element is read-only, deduced from the selection above, rather than a selection itself.
- "Change how the control appears on your dashboard" -> Present tense

Reply:
Thank you so much! These copy changes sound great - fixed in 79297e7a4797e3cb90ed7092e09d23805e1f35d1 :bow: